src/css/style.css

```diff
@@ -0,0 +1,85 @@
+body {
+    min-height: 80%;
+    padding-top: 50px;
+    background-color: #e1e8f0;
+    font-family: "Oswald", sans-serif;
+}
+
+h1 {
+    text-align: center;
+    font-size: 1.8em;
+}
+
+.row {
+    background-color: #e1e8f0;
+}
+
+.col-xs-12 {
+    background-color: #fff;
+}
+
+.navbar-default {
+    background-color: #9068be;
+    border-color: #E7E7E7;
+}
+
+.navbar-default .navbar-brand {
+    color: #fff;
+}
+
```

Comment:
Unnecessary spaces. You can use some text editor plugins like beautify to fix this